packages/cli/src/commands/deprecated/test.ts

```diff
@@ -35,15 +35,18 @@ const commandModule: CommandModule<Options, Options> = {
       ...(forgeOpts?.split(" ") || []),
     ]);
 
-    // Reset LibDeploy.sol
-    console.log("Reset LibDeploy.sol");
-    await resetLibDeploy(testDir);
-
-    process.on("SIGINT", () => {
+    process.on("SIGINT", async () => {
       console.log("\ngracefully shutting down from SIGINT (Crtl-C)");
+      await resetLibDeploy(testDir);
       child.kill();
       process.exit();
     });
+
+    // Reset LibDeploy.sol
+    console.log("Reset LibDeploy.sol");
+    await resetLibDeploy(testDir);
+
+    await child;
```

Comment:
```suggestion
    await child;
    
    // Reset LibDeploy.sol
    console.log("Reset LibDeploy.sol");
    await resetLibDeploy(testDir);
```

Does this need to wait until after the tests run before resetting the dir?

Reply:
right, I got confused with the refactor and broke it again haha